README.md

```diff
@@ -1,40 +1,72 @@
 [![Open Source Helpers](https://www.codetriage.com/habitat-sh/habitat/badges/users.svg)](https://www.codetriage.com/habitat-sh/habitat)
 
-Want to try Habitat? [Get started here](https://www.habitat.sh/learn/).
 
-Habitat is an application automation framework that allows you to build applications that have automation built-in. This provides modern applications that:
+[Habitat](http://habitat.sh) is open source software that creates platform-independent build artifacts and provides built-in deployment and management capabilities.
 
-* Provide repeatable builds
-* Run from single, immutable assets
-* Allow for runtime configuration for multiple deployment scenarios
-* Are agnostic to operating environment (works the same on bare metal, virtualization, containers, PaaS)
-* Provide idempotent behavior (the same inputs to the same asset provide the same outcome)
-* Provide convergent behavior (each service makes progress towards the correct behavior in the face of failure)
-* Expose promises to those who rely on it
-* Provide a low barrier to entry
-* Are language agnostic
+The goal of Habitat is to allow you to automate your application behavior when you create your application, and then bundle your application with the automation it needs to behave with the correct run time behavior, update strategies, failure handling strategies, and scaling behavior, wherever you choose to deploy it.
 
-To learn more about Habitat, please visit the [Habitat website](https://www.habitat.sh).
+See a quick demo of how to build, deploy and manage an application with Habitat:
 
-Participation in the Habitat community is governed by the [code of conduct](https://github.com/habitat-sh/habitat/blob/master/CODE_OF_CONDUCT.md).
+[![Build, Deploy and Manage with Habitat (5:33)](images/overview-youtube-image.jpg)](http://www.youtube.com/watch?v=VW1DwDezlqM)
 
 # Table of Contents
+* [Infographics](#infographics)
+* [Hands-on Demos](#hands-on-demos)
 * [Install](#install)
 * [Contribute](#contribute)
 * [Documentation](#documentation)
-* [Code Organization](#repo-organization)
+* [Code Organization](#code-organization)
 * [Roadmap](#roadmap)
 * [Community and support](#community-and-support)
 * [Building](#building)
 * [Further reference material](#further-reference-material)
+* [Code of Conduct](#code-of-conduct)
 * [License](#license)
 
+## Infographics
+Graphics that will help you and your team better understand the concepts and how they fit together into the larger Habitat ecosystem.
+### Where Habitat Fits
+
+[![Habitat Flow Infographic](images/habitat-flow-infographic.png)](http://habitat.sh)
+
+[//]: # (Try the interactive infographics on the [website](http://habitat.sh)
+
+### How Habitat Works
+* [Architecture Overview](https://www.habitat.sh/images/infographics/habitat-architecture-overview-7e40c4b3.png)
+* [**Initial Package Build** Flow](https://www.habitat.sh/images/infographics/habitat-initial-package-build-flow-43ccb287.png)
```

Comment:
I missed it earlier, but do these items need to be bolded for some reason? I might suggest they just be regular text, since they're already linked, and the bolding doesn't seem to be especially disambiguating.

Reply:
I agree with you. The bold emphasis doesn't make much sense after I removed some links and groups.